util/testutil/workers/containerd.go

```diff
@@ -266,9 +277,13 @@ func runStargzSnapshotter(cfg *integration.BackendConfig) (address string, cl fu
 	if err != nil {
 		return "", nil, err
 	}
-	if err = integration.WaitUnix(address, 10*time.Second, cmd); err != nil {
+	if err = integration.WaitSocket(address, 10*time.Second, cmd); err != nil {
 		snStop()
-		return "", nil, errors.Wrapf(err, "containerd-stargz-grpc did not start up: %s", integration.FormatLogs(cfg.Logs))
+		errMsg := fmt.Sprintf(
```

Comment:
`errors.Wrapf()` can handle formatting on it's own. There is no need to format the error with `fmt.Sprintf()` and pass it to `errors.Wrapf()`. Please do not make changes just for the sake of aesthetics.

Reply:
reverted.